spec/models/validations/sales/setup_validations_spec.rb

```diff
@@ -119,15 +119,15 @@
         it "cannot create new logs for the archived collection year" do
           record.saledate = Time.zone.local(2023, 1, 1)
           setup_validator.validate_saledate_collection_year(record)
-          expect(record.errors["saledate"]).to include(match "Enter a date within the 23/24 or 24/25 collection years, which is between 1st April 2023 and 31st March 2025")
+          expect(record.errors["saledate"]).to include(match "Enter a date within the 23/24 or 24/25 collection years, which is between 1st April 2023 and 31st March 2025.")
         end
 
         it "can edit already created logs for the previous collection year" do
           record.saledate = Time.zone.local(2024, 1, 2)
           record.save!(validate: false)
           record.saledate = Time.zone.local(2024, 1, 1)
           setup_validator.validate_saledate_collection_year(record)
-          expect(record.errors["saledate"]).not_to include(match "Enter a date within the 24/25 collection year, which is between 1st April 2024 and 31st March 2025")
+          expect(record.errors["saledate"]).not_to include(match "Enter a date within the 24/25 collection year, which is between 1st April 2024 and 31st March 2025.")
```

Comment:
There are a couple of tests like this which wouldn't have failed with the removal of `.` . This one has `match` so would probably still be testing correctly, but I checked for others and updated the few that I found. 
We mostly seem to test absence with `to be_empty` etc

Reply:
Yeah, this seems the biggest risk so glad to know you've checked! I would expect us mostly to be using `to be_empty` though 🤞.